packages/eslint-config/eslintrc.js

```diff
@@ -93,6 +93,9 @@ module.exports = {
     '@typescript-eslint/no-empty-interface': 'off',
     '@typescript-eslint/no-non-null-asserted-optional-chain': 'error',
 
+    // Disable warning  Missing return type on function for now
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
Please open a follow-up issue to investigate how can we enable this rule in the future.

Reply:
See https://github.com/strongloop/loopback-next/issues/5532